src/components/selectEvent/HexathonModal.tsx

```diff
@@ -3,8 +3,14 @@ import {Button, Modal, ModalOverlay, ModalContent, ModalHeader, ModalCloseButton
 import { useForm } from "react-hook-form";
 import axios from "axios";
 import { apiUrl, Service } from "@hex-labs/core";
+import { EditIcon } from "@chakra-ui/icons";
 
-const HexathonModal: React.FC = () => {
+interface Props {
+  num: number;
```

Comment:
Can we make this field more descriptive of its function? naming it something like `action` would make it more readable, and it would take `CREATE` or `EDIT`.